public/about.html

```diff
@@ -2,17 +2,43 @@
 
 <html lang="en">
 
+<head>
+    <title>RERUM Geolocator</title>
+    <meta charset="UTF-8">
+    <link rel="icon" href="/images/favicon.ico" type="image/ico">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://unpkg.com/chota@latest">
+    <link rel="stylesheet" href="stylesheets/style.css">
+    <script src="components/layout.js"></script>
+</head>
+
+<link rel="stylesheet" href="https://unpkg.com/chota@latest">
+<link rel="stylesheet" href="stylesheets/style.css">
```

Comment:
You already have these in the `<head>` element.  These are duplicates and should be removed.